<?xml version="1.0"?>

<launch>

<param name="port" value="/dev/ttyACM2" />
<node pkg="joystic_wheelchair" type="send_voltage_to_arduino" name="send_voltage_to_arduino"/>

</launch>
